guides/common/modules/con_overview-of-project-mcp-integration.adoc

```diff
@@ -0,0 +1,9 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="overview-of-project-mcp-integration"]
+= Overview of {Project} MCP integration
+
+{Project} provides a container image that you can use to run an MCP server locally.
+The {Project} MCP server can generate reports based on {Project} inventory.
+You can connect your MCP client to the {Project} MCP server to enable it to extract and process this inventory information.
+By querying the AI application through the MCP client, you can generate dynamic and comprehensive reports from your {Project} environment.
```

Comment:
I'd emphasize that the MCP server has a read-only access to sat, so it can't make changes in there. Also I'd make note that the mcp server is meant to be "deployed" on a per user basis, meaning that everyone should deploy their own and use their own credentials